delfin/common/constants.py

```diff
@@ -302,9 +301,25 @@ class ResourceSync(object):
     FAILED = 101
 
 
-class Task(object):
-    DEFAULT_TASK_INTERVAL = 30
-    """Default task interval"""
-    PERFORMANCE_TASK_METHOD = "delfin.task_manager.tasks.telemetry." \
-                              "PerformanceCollectionTask"
+class TelemetryCollection(object):
     """Performance monitoring task name"""
+    PERFORMANCE_TASK_METHOD = "delfin.task_manager.scheduler.schedulers." \
+                              "telemetry.performance_collection_handler." \
+                              "PerformanceCollectionHandler"
+    """Performance monitoring job interval"""
+    PERIODIC_JOB_INTERVAL = 180
+    """Failed Performance monitoring job interval"""
+    FAILED_JOB_SCHEDULE_INTERVAL = 240
+
+
+class TelemetryTaskStatus(object):
+    """Telemetry task enum"""
+    TASK_EXEC_STATUS_SUCCESS = True
```

Comment:
How about timeout status

Reply:
Actually the status is not related to RPC timeout. Its used for the scenario where RPC task need to return status of task execution. The called received the response and make decision over it.

Reply:
Currently this scenario is not handled, will be tracked through https://github.com/sodafoundation/delfin/issues/525